### DISCUSSION TITLE
Supports, anything but good :-(

### DISCUSSION THREAD

**Discussion by hpoperator** [supermerill/SuperSlicer#3781] [Q&A]
Hello dear community,
I have learned to appreciate and love SuperSlicer in the last few days. The adjustment possibilities with it are phenomenal (sorry, only knew Cura 🤷‍♂️ before). Thanks to Ellis3dp.com my prints are also good. 

But, and I have to get this off my chest, the support, no matter how you set it up, is in comparison to Cura (and I don't mean the tree structure (which the SuperSlicer developer has been trying to integrate for almost 4 years)). My flow is now set accurately, but no matter how I set the supports, they stick to my part later, or if I enter a slightly larger distance, they still stick, but the supported area also gets messy (sagging).

Also have observed earlier that it generates the supports so that they hang in the air in places, so the supports are not supported. I have also tried to paint supports myself, but the result is more or less the same.
I am afraid to use supports every time.

Example:
In Cura, only the holes are supported. Sopports are not necessarily pulled down to the ground ("Everywhere" is enabled) just to support the one hole.

Also a screenshot of the support without support.

![2023-04-05 17_24_04-_Unbenannt - SuperSlicer_2 4  based on PrusaSlicer](https://user-images.githubusercontent.com/94447783/230133315-4c94daa6-98d7-44f9-9095-07e89b264bcf.png)
![2023-04-05 17_22_28-_Unbenannt - SuperSlicer_2 4  based on PrusaSlicer](https://user-images.githubusercontent.com/94447783/230133322-d79f1776-3200-4399-bb86-46207f2a716c.png)
![2023-04-05 17_23_19-_Unbenannt - SuperSlicer_2 4  based on PrusaSlicer](https://user-images.githubusercontent.com/94447783/230133323-d4054053-4d0a-4d98-a845-83634dbb95f4.png)

Currently print exclusively ASA (Dura_Pro from Extrudr).

here as an example the support creation in cura:

![2023-04-05 19_22_14-CE3PRO_12_DinRail_LRS350 - UltiMaker Cura](https://user-images.githubusercontent.com/94447783/230157217-90241c73-39e1-48be-ad06-b433819574db.png)
![2023-04-05 19_21_29-CE3PRO_xboxonecontrollerbackv5 - UltiMaker Cura](https://user-images.githubusercontent.com/94447783/230157346-cf79a79d-dba8-4934-a1a0-92106c175249.png)

there really exists a need for action at superslicer in terms of programming!

Is it running better for you guys?

Any advice would be greatly appreciated.

Many thanks, and best regards

**Comment by neophyl** [comment]
Looks like you are using 'classic' supports, you need to switch to 'snug' to have them act more like the ones pictured from Cura.  Please attach a zipped up project file that can be examined (also what version you are using etc) then maybe someone can take a look.
